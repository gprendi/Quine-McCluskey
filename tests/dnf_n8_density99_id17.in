d 8
11111110
11010100
00010111
11001000
11100101
11110111
10101000
01011000
01001000
01101011
10001011
10101001
10011100
01101000
11001001
10001010
01000101
01100011
01010011
10100100
01001001
01011010
01110001
11100111
00100100
00001100
10000101
00111111
00100010
10101010
00000101
00001001
00010100
10111010
10010010
01111110
10111101
01011011
10000001
00101000
00001111
10000000
00011100
01010111
00100011
10011111
00010011
11101010
01010110
00011001
11110001
10011011
01011110
00101110
00011111
11110101
11000101
00110100
01101110
00101001
01001100
10111111
00101111
01001101
01010100
11011011
11100011
00111010
10010110
11011001
10000100
01001111
10010011
10010001
10000111
10011001
10010000
10101111
11101111
01101010
01101111
10100101
11000011
10001111
01110101
00100111
10101100
10111001
10100111
10001101
11100001
00100000
00000010
11111001
01110000
01111101
11011000
00001010
01001010
01010101
01011101
10111011
11011101
00010010
01000111
00111001
11101101
11001100
01001110
10101110
01010010
10110101
10001001
00011011
11010000
00110001
11010011
00111101
01110011
00101100
01110111
10111110
10111100
01000000
11001111
10010100
11101000
11100110
01111001
11111101
00111000
00111011
11001110
00011000
11000110
01101001
01011111
10111000
01100111
11011100
00000011
01000011
10100011
11100010
00100001
11000000
01111111
01010000
11110010
01111011
11001011
10110001
00110101
01100001
00010000
10000010
10110111
00110011
11011111
11000111
01111100
00110110
11111100
01011100
11010010
10001110
11110000
11011110
10101101
01000100
11000010
11001101
10011010
00011110
00000000
11011010
11111011
10110011
00010110
01110100
10011110
11010110
01100110
00001110
01101101
11101100
10110110
00000111
10001100
10101011
10110100
11100000
00110111
00010101
10010101
00100110
10100010
01001011
00111110
01100010
01100101
10000110
00110010
01000001
11111000
00101011
00101101
10001000
01111000
00111100
01100000
00011101
00000001
00011010
10110010
00010001
11110011
00000100
11101110
11111010
11101011
00001011
10110000
01110010
10100110
01000010
11000100
00110000
11001010
01110110
01010001
01111010
11000001
11010001
01101100
10011000
11111111
10010111
10000011
01011001
00001000
11010101
00000110
00100101
00101010
11110100
11110110
00001101
11100100
10100001
11010111
10011101
10100000
