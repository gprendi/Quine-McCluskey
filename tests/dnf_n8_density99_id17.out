p dnf 8 0
-1 5 0
-1 -6 0
-1 8 0
1 -3 0
1 -5 0
1 6 0
1 7 0
1 -8 0
-2 0
-3 5 0
-3 -6 0
-3 -7 0
-3 8 0
3 7 0
4 0
-5 -6 0
-5 8 0
5 6 0
5 7 0
5 -8 0
-6 7 0
-6 -8 0
6 8 0
7 8 0
